docs/developer-guide.md

```diff
@@ -382,3 +290,92 @@ Services that are objects (rather than numbers, strings, or functions) may need
 For now we don't use the `@api` annotation as in OpenLayers. We explicitly use
 `goog.exportProperty` in separate "exports" files. See the `exports` directory
 in the repo.
+
+### Running examples
+
+#### Local mode
+
+To run the examples locally, just run `make serve` and open your browser in
+`http://localhost:3000/examples/` uri.
+
+
+All extern javascript includes must be defined in the example file, explicitly.
+
+```html
+  <script src="../node_modules/jquery/dist/jquery.js"></script>
+  <script src="../node_modules/angular/angular.js"></script>
+  <script src="../node_modules/bootstrap/dist/js/bootstrap.js"></script>
+```
+
+To include the example code and all its dependencies through `ngeo` and `ol3`
+you have to include the following tag:
+
+```html
+  <script src="/@?main=scaleselector.js"></script>
+```
+
+The `/@?main=scaleselector.js` will take the `scaleselector.js` file in the
+example folder and will load in debug mode all the dependency tree built with
+the `goog.require()` call.
+In local examples, you will have all javascripts files in debug mode of ol3 , 
+ngeo, contribs and goog files loaded.
+
+#### Examples in gh-pages
+
+When you execute the target `gh-pages`, the makefile creates a folder 
```

Comment:
The target `examples-hosted` to create the `.build/examples-hosted`, to test it locally, and the target `gh-pages` to upload it to GitHub pages ...